tests/views/tp.py

```diff
@@ -53,15 +53,15 @@ def _test_browse_view(tp, request, response, kwargs):
     pootle_path = "%s%s" % (tp.pootle_path, resource_path)
 
     if not (kwargs["dir_path"] or kwargs.get("filename")):
-        ob = tp.directory
```

Comment:
...or where it can be objects of totally different sorts

feel free to ignore  - its a nit on a test